src/base/_fonts.scss

```diff
@@ -0,0 +1,49 @@
+@mixin _non-variable-font-face($name, $path, $parameter) {
```

Comment:
I'm curious if the leading underscores are meant to signify something?

Reply:
Yes, it is used to make the mixin not exported from that file

https://sass-lang.com/documentation/at-rules/use#private-members